python/pyspark/sql/tests/test_arrow.py

```diff
@@ -30,10 +30,14 @@
     pandas_requirement_message, pyarrow_requirement_message
 from pyspark.testing.utils import QuietTest
 from pyspark.util import _exception_message
+from distutils.version import LooseVersion
 
 if have_pandas:
     import pandas as pd
     from pandas.util.testing import assert_frame_equal
+    pandas_version = LooseVersion(pd.__version__)
+else:
+    pandas_version = LooseVersion("0")
```

Comment:
I would do something like:

```python
pandas_version = None
if have_pandas:
    import pandas as pd
    from pandas.util.testing import assert_frame_equal
    pandas_version = LooseVersion(pd.__version__)
```

if the linter is not happy with it.

Reply:
Nice, will update